LICENSE

```diff
@@ -1,6 +1,6 @@
 vertica-python
 
-Copyright (c) 2013 Uber Technologies, Inc.
+Copyright (c) 2013-2017 Uber Technologies, Inc.
```

Comment:
2018?

Reply:
Legal team has given this instruction so I think this should be fine.